travis.sh

```diff
@@ -4,10 +4,11 @@ set -euo pipefail
 
 function configureTravis {
   mkdir -p ~/.local
-  curl -sSL https://github.com/SonarSource/travis-utils/tarball/v38 | tar zx --strip-components 1 -C ~/.local
+  curl -sSL https://github.com/SonarSource/travis-utils/tarball/v43 | tar zx --strip-components 1 -C ~/.local
   source ~/.local/bin/install
 }
 configureTravis
+source ~/.local/bin/installMaven35
```

Comment:
@alban-auzeill what is this for?

Reply:
There's two commits, this change is part of "Update travis-utils from v38 to v43"
travis-utils v38 was not working, and v43 need "installMaven35" to not raise the following error:
```
[ERROR] No plugin found for prefix 'sonarsource' in the current project and in the plugin groups [org.apache.maven.plugins, org.codehaus.mojo] available from the repositories
```